package.json

```diff
@@ -566,7 +566,7 @@
           "*Resource*Pack*/animations/**/*.{json,jsonc,json5}",
           "*RP*/animations/**/*.{json,jsonc,json5}",
           "*rp*/animations/**/*.{json,jsonc,json5}",
-          "*.{animation.rp,animation,anim.rp,a.rp,rpa}.{json,jsonc,json5}"
+          "*.{animation.rp,animation,anim.rp,a.rp,rpa,animation}.{json,jsonc,json5}"
```

Comment:
animation is already there, item nr 2 😛 
I added it when you mentioned it couple of days ago 😄

Reply:
I'm blind :(

Reply:
Nw 😄